code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1185,6 +1190,89 @@
 		M.confused = min(M.confused + 2, 5)
 	..()
 
+/datum/reagent/carbon_monoxide
+	name = "Carbon Monoxide"
+	description = "A highly dangerous gas for sapients."
+	reagent_state = GAS
+	metabolization_rate = 0.1 * REAGENTS_METABOLISM
```

Comment:
0.04 metabolization speed means it'll likely need to be purged externally if you breathe any, and while injecting reagent oxygen might be intuitive for people who know how it works its probably not intuitive for a player

Reply:
Yeah
Probably going to make it a normal rate, this was from when I couldn't get it to disappear immediately

Reply:
changed